### PR TITLE
Add information about the putative origin of SARS-CoV-2.

### PR DESCRIPTION
**Pull request by cbrueffer**

### CODE REVIEW

**Comment by cbrueffer** [comment]
References the paper suggested in https://github.com/greenelab/covid19-review/issues/41

File: content/02.introduction.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ This is a real-time, collaborative effort that welcomes submissions from scienti
 Coronaviruses are RNA viruses that...
 [Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important]
 
+The origin of the SARS-CoV-19 virus is not yet fully understood.
+Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-19 is unlikely to have originated from a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi: 10.1038/s41591-020-0820-9].
+Among known coronaviruses, SARS-CoV-19 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi 10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi 10.1101/2020.02.19.950253].
```
Comment (cgreene) [inline content/02.introduction.md]:
```suggestion
Among known coronaviruses, SARS-CoV-19 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi:10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253].
```

File: content/02.introduction.md [hunk 2]
```diff
@@ -27,6 +27,11 @@ This is a real-time, collaborative effort that welcomes submissions from scienti
 Coronaviruses are RNA viruses that...
 [Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important]
 
+The origin of the SARS-CoV-19 virus is not yet fully understood.
+Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-19 is unlikely to have originated from a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi: 10.1038/s41591-020-0820-9].
```
Comment (cgreene) [inline content/02.introduction.md]:
```suggestion
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-19 is unlikely to have originated from a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
```

File: content/02.introduction.md [hunk 3]
```diff
@@ -27,6 +27,11 @@ This is a real-time, collaborative effort that welcomes submissions from scienti
...
+Among known coronaviruses, SARS-CoV-19 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi 10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi 10.1101/2020.02.19.950253].
+This suggests that SARS-CoV-19 may have originated in viral reservoirs of similar hosts, however current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi 10.1038/s41591-020-0820-9].
```
Comment (cgreene) [inline content/02.introduction.md]:
```suggestion
This suggests that SARS-CoV-19 may have originated in viral reservoirs of similar hosts, however current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].
```

Comment (cbrueffer) [reply content/02.introduction.md]: Thanks a lot, I've fixed the tags accordingly.

**Pull request merged**